src/components/Area/movie-area.js

```diff
@@ -10,6 +10,7 @@ const Area = ({ movies }) => {
       img= {movie.poster_path}
       key= {movie.id}
       id= {movie.id}
+      title={movie.title}
```

Comment:
Excellent job catching this with your wave extension. Didn't realize our alt text was undefined without you spotting it.